src/components/SEO/index.tsx

```diff
@@ -20,10 +20,7 @@ const SEO: React.FC = () => {
       {/* title to display for the specific link being shared */}
       <meta property="og:title" content="SPACE 2022" />
```

Comment:
Is this supposed to be changed too

Reply:
Oops good catch, I just updated the SEO stuff